bravado/http_future.py

```diff
@@ -200,6 +234,7 @@ def response(self, timeout=None, fallback_result=SENTINEL, exceptions_to_catch=F
         )
 
     def result(self, timeout=None):
+        # type: (typing.Optional[float]) -> typing.Union[T, typing.Tuple[T, IncomingResponse]]
```

Comment:
It can also be just `IncomingResponse` (the last of the three `return` statements). It's strange that mypy didn't catch this...

Reply:
nice catch